src/V3Broken.cpp

```diff
@@ -270,8 +281,30 @@ class BrokenCheckVisitor final : public AstNVisitor {
             if (const AstNodeDType* dnodep = VN_CAST(nodep, NodeDType)) checkWidthMin(dnodep);
         }
         checkWidthMin(nodep);
+    }
+    void processExit(AstNode* nodep) { BrokenTable::setUnder(nodep, false); }
+    void processAndIterate(AstNode* nodep) {
+        processEnter(nodep);
         iterateChildrenConst(nodep);
-        BrokenTable::setUnder(nodep, false);
+        processExit(nodep);
+    }
+    void processAndIterateList(AstNode* nodep) {
+        do {
+            processAndIterate(nodep);
+            nodep = nodep->nextp();
+        } while (nodep);
```

Comment:
```suggestion
        while (nodep) {
            processAndIterate(nodep);
            nodep = nodep->nextp();
        }
```

Seems harmless and safer